great_expectations/datasource/fluent/data_asset/data_connector/azure_blob_storage_data_connector.py

```diff
@@ -41,6 +49,13 @@ class AzureBlobStorageDataConnector(FilePathDataConnector):
         file_path_template_map_fn: Format function mapping path to fully-qualified resource on ABS
     """
 
+    asset_level_option_keys: ClassVar[tuple[str, ...]] = (
```

Comment:
@Kilo59 Confirming that I am seeing it correctly that we still have these entities defined in each specific DataConnector but used only elsewhere, right?  Thanks.

Reply:
Yes it's used elsewhere in order to pluck out the data_connector specific args from the `kwargs` passed to `add_<ASSET_TYPE>_asset(name: str, **kwargs)`